src/main/scala/gestalt/api.scala

```diff
@@ -122,8 +143,26 @@ object api extends Toolbox { pkg =>
   }
 
   implicit class UntypedTermTreeOps(tree: TermTree) {
-    def select(name: String): TermTree = Select(tree, name)
     def appliedTo(args: TermTree*): TermTree = Apply(tree, args.toList)
+    def selectType(path: String): TypeTree = {
+      val parts = path.split('.')
+
+      val prefix = parts.init.foldLeft[TermTree](tree) { (prefix, name) =>
+        prefix.select(name)
+      }
+
+      TypeSelect(prefix, parts.last)
```

Comment:
This should be a fold over `TypeTree` not `TermTree`.
It is possible to eliminate the recursive calls if we handle  the `head` specially not the `last`.
```scala
parts.tail.foldLeft[TypeTree](TypeSelect(tree,parts.head)){
  (prefix, name) =>
    TypeSelect(prefix, name)
}
````